src/rgw/rgw_rest_s3.cc

```diff
@@ -586,15 +588,11 @@ void RGWPutObjTags_ObjStore_S3::send_response()
 
 void RGWDeleteObjTags_ObjStore_S3::send_response()
 {
-  int r = op_ret;
-  if (r == -ENOENT)
-    r = 0;
-  if (!r)
-    r = STATUS_NO_CONTENT;
-
-  set_req_state_err(s, r);
+  if (op_ret)
+    set_req_state_err(s, op_ret);
```

Comment:
originally this call is not under any condition, why restricting now ?

Reply:
It is ignoring ENOENT by making r=0.

Reply:
> I feel ENOENT should be ignored for delete operations.

@sjain09 why?

@selvakumaar5496 can you revert to Saurabh's comment, behavior on AWS ?

Reply:
Yeah, Sure

Reply:
As i understood after our discussion, its not changing any api response behavior for Rados and correcting for Motr
Confirm AWS behavior still, and even if its different, we wont align to that, given required changes would be beyond Motr SAL Layer

Reply:
AWS does not ignore ENOENT too.. 

[root@ssc-vm-g2-rhev4-3107 ~]# aws s3api delete-object-tagging --bucket test-buckets123434 --key test-key-02

An error occurred (NoSuchKey) when calling the DeleteObjectTagging operation: The specified key does not exist.